tests/Platform/QueryResultTypeWalkerFetchTypeMatrixTest.php

```diff
@@ -630,3803 +630,3803 @@ public static function provideCases(): iterable
 	{
 		yield ' -1' => [
 			'data' => self::dataDefault(),
-			'select' => 'SELECT -1 FROM %s t',
-			'mysql' => new ConstantIntegerType(-1),
-			'sqlite' => new ConstantIntegerType(-1),
-			'pdo_pgsql' => new ConstantIntegerType(-1),
-			'pgsql' => new ConstantIntegerType(-1),
-			'mssql' => self::mixed(),
-			'mysqlResult' => -1,
-			'sqliteResult' => -1,
-			'pdoPgsqlResult' => -1,
-			'pgsqlResult' => -1,
-			'mssqlResult' => -1,
+			'dqlTemplate' => 'SELECT -1 FROM %s t',
```

Comment:
I think the error which got triggered by the wrong arg names should be made phpunit11+ only